src/commands/Minion/mine.ts

```diff
@@ -97,15 +101,17 @@ export default class extends BotCommand {
 
 		// For each pickaxe, if they have it, give them its' bonus and break.
 		const boosts = [];
-		if (msg.author.skillLevel(SkillsEnum.Mining) >= 61) {
-			for (const pickaxe of pickaxes) {
-				if (msg.author.hasItemEquippedOrInBank(pickaxe.id)) {
-					timeToMine = Math.floor(timeToMine * ((100 - pickaxe.reductionPercent) / 100));
-					boosts.push(`${pickaxe.reductionPercent}% for ${itemNameFromID(pickaxe.id)}`);
-					break;
-				}
+		for (const pickaxe of pickaxes) {
+			if (
+				msg.author.hasItemEquippedOrInBank(pickaxe.id) &&
+				msg.author.skillLevel(SkillsEnum.Mining) >= pickaxe.miningLvl
+			) {
+				timeToMine = Math.floor(timeToMine * ((100 - pickaxe.reductionPercent) / 100));
```

Comment:
can you use `reduceNumByPercent` for the reduction (and anywhere else), as a little cleanup